maas/client/viscera/machines.py

```diff
@@ -174,6 +177,8 @@ class Machine(Object, metaclass=MachineType):
         :param hwe_kernel: The HWE kernel to deploy. Probably only relevant
             when deploying Ubuntu.
         :param comment: A comment for the event log.
+        :param wait: If specified, wait until the deploy is complete, polling
+            at the specified frequency.
```

Comment:
Specifying the total wait time doesn't make sense does it? This is managed by MAAS